__tests__/app.test.js

```diff
@@ -52,4 +52,37 @@ describe('APP', () => {
             })
         });
     });
+    describe('GET /api/articles/:article_id', () => {
+        test('should repsond with a status 200 and an object with the appropriate keys', () => {
+            return request(app)
+            .get('/api/articles/3')
+            .expect(200)
+            .then((response) => {
+                expect(response.body.article).toHaveProperty('author', expect.any(String))
+                expect(response.body.article).toHaveProperty('title', expect.any(String))
+                expect(response.body.article).toHaveProperty('article_id', expect.any(Number))
```

Comment:
When testing the article_id in this case do we expect it to be **_any_** number>

Reply:
Ah yes, this should just be the same as I'm testing for. I've now changed this.